cmd/uid_provider/api/server.go

```diff
@@ -0,0 +1,14 @@
+package api
+
+import (
+	ctrl "gitlab.inspr.dev/inspr/core/cmd/uid_provider/api/controllers"
```

Comment:
change the import name to api?

Reply:
Will do, but I got this reference from how it's implemented in Insprd